src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -200,6 +201,16 @@ public Builder setRecordNumber(final long recordNumber) {
             return asThis();
         }
 
+        /**
+         * Sets the Charset to use for the reader.
```

Comment:
"Sets the character encoding to be used for the reader"? Your original comment is not very clear.